.github/workflows/rotki_sqldiff.yml

```diff
@@ -0,0 +1,31 @@
+name: SQLDiff
+
+on:
+  pull_request_target:
+    branches:
+      - main
+
```

Comment:
is this correct? Shouldn't it be develop/bugfixes/master?

Reply:
True